app/controllers/start_group_controller.rb

```diff
@@ -2,7 +2,9 @@ class StartGroupController < ApplicationController
   def new
     @errors = []
     @group = Group.new
-    if current_user.present? && ENV['NEW_GROUPS_ON_ANGULAR']
```

Comment:
I think the enable_angular method shows the screen we want to bypass, so don't we just want redirect them to dashboard with the modal open if angular is enabled?